compose/galaxy-postgres/init-galaxy-db.sql.in

```diff
@@ -8237,7 +8126,7 @@ GalaxyTools	lib/tool_shed/galaxy_install/migrate	1
 --
 
 COPY migrate_version (repository_id, repository_path, version) FROM stdin;
-Galaxy	lib/galaxy/model/migrate	144
```

Comment:
My bad, I must have been testing a few things on 19.01 when I pushed this... sorry.

Reply:
I have been thinking for a while on incorporating

https://github.com/bgruening/docker-galaxy-stable/blob/dev/compose/dumpsql.sh#L9-L36

inside the build-orchestration-images script, right after the init container creation (using that init container created). Then this whole file would be generated on the fly when building the containers, based on the version that it was used of Galaxy for those containers (so fairly failure-proof). Would you consider adding that to your PR @abretaud ?

Reply:
Yep, that would be cool. I'm not sure how to do it properly though: do you mean that at the end of build-orchestration-images we should:
1) run a docker-compose with an empty postgresql database
2) run the install_db.sh in galaxy-web
3) run the dumpsql.sh script
4) (re)build the galaxy-postgres image

Not sure I'll have time to write this now, maybe it could be another PR?

Reply:
Yes, don't worry, I'll PR it soon.